packages/apps/docs/src/components/Markdown/UnorderedList/UnorderedList.tsx

```diff
@@ -1,11 +1,11 @@
-import { StyledList } from './styles';
+import { ulListClass } from './styles.css';
 
-import React, { FC } from 'react';
+import React, { FC, ReactNode } from 'react';
 
 interface IProp {
-  children: string;
+  children: ReactNode;
```

Comment:
Instead of switching from string to any React Node you could make this something like:

```
FunctionComponentElement<HTMLOListElement | HTMLUListElement>[]
```